packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -301,14 +427,14 @@ abstract class ResidentRunner {
         return true;
       }
     } else if (lower == 's') {
-      if (device.supportsScreenshot) {
-        await _screenshot();
-        return true;
+      for (RunningDevice device in runningDevices) {
+        if (device.device.supportsScreenshot)
+          await _screenshot(device);
       }
+      return true;
     } else if (lower == 'o') {
       if (supportsServiceProtocol && isRunningDebug) {
-        final String platform = await _debugRotatePlatform();
-        print('Switched operating system to: $platform');
```

Comment:
This is a regression. It's very important (especially in the single-device case) to report what operating system we've switched to.